src/core/json/jsonpath_test.cc

```diff
@@ -165,35 +165,88 @@ TEST_F(JsonPathTest, Path) {
   int called = 0;
 
   // Empty path
-  EvaluatePath(path, json, [&](const JsonType& val) { ++called; });
+  EvaluatePath(path, json, [&](optional<string_view>, const JsonType& val) { ++called; });
```

Comment:
we pass key/value pair now. key is nullopt for array elements that do not have a key

Reply:
can it be empty string_view?

Reply:
Json key can be empty